R/modMAtree.R

```diff
@@ -645,6 +648,39 @@ modMAtree <- function(MApopdat,
 	ifelse(MAmethod == "gregEN", "gregElasticNet", 
 	ifelse(MAmethod == "ratio", "ratioEstimator", "horvitzThompson"))))
   message("generating estimates using mase::", masemethod, " function...\n")
+  
+  ## do modelselect here?
+  if (MAmethod == "greg" && modelselect == T) {
```

Comment:
before we merge this we want to get a sense for how different the estimates are when modelselection is running and also see how often different variables are being chosen. (compare to running model select at each condition level in the sample)